README.md

```diff
@@ -69,7 +69,12 @@ sudo snap install natls
 ```bash
 sudo snap refresh natls
 ``` 
-
+#### Another alternative (arch linux)
+[nat](https://aur.archlinux.org/packages/nat/)
+```bash
+paru -S nat
```

Comment:
Replace with yay -S nat and make it (or any AUR helper) because most people use yay. Paru >>>> but yay is just more popular.

Reply:
meh